practice-app/server/src/server.js

```diff
@@ -26,6 +27,37 @@ mongoose.connection.useDb("practice");
 
 const PORT = process.env.API_PORT;
 const server = http.createServer(app);
+const DB = process.env.DB;
+const AUTH0_DOMAIN = process.env.auth0_domain
+// var KEYS =[];
+
+const connectDB = async () => {
+    await mongoose
+        .connect(DB, {
+            useNewUrlParser: true,
+            useUnifiedTopology: true,
+        })
+        .then(() => {
+            console.log("DB Connection established");
+            var connection = mongoose.connection;
+            connection.db.listCollections().toArray(function(err, names) {
+              if (err) {
+                  console.log(err);
+              }
+              else {
+                  names.forEach(function(e,i,a) {
+                      console.log("--->>", e.name);
+                  });
+              }
+            });
+        })
+        .catch((e) => {
+            console.log(e);
+        });
+
+};
```

Comment:
DB connection codes should have been pulled from the main branch. This is creating a conflict. Please resolve it.